dns.go

```diff
@@ -7,6 +7,9 @@ const (
 	defaultTtl = 3600    // Default internal TTL.
 
 	// DefaultMsgSize is the standard default for messages larger than 512 bytes.
+	// RFC6891 6.2.3 recommends 1280-1410 bytes for IP (v4 or v6) over ethernet.
```

Comment:
this size pertains to fragmentation, not truncation. This comment could indeed be better.
~~~
 DefaultMsgSize is  a default for messages larger than 512 bytes.
 RFC6891 6.2.3 recommends 1280-1410 bytes for IP (v4 or v6) over ethernet.
~~~

Reply:
Thanks, I've rewritten it to include both recommendations from 6.2 in the RFC.